packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -423,65 +669,168 @@ class FocusScopeNode extends Object with DiagnosticableTreeMixin {
 ///  * [FocusScopeNode], which is an interior node in the focus tree.
 ///  * [FocusScope.of], which provides the [FocusScopeNode] for a given
 ///    [BuildContext].
-class FocusManager {
+class FocusManager with DiagnosticableTreeMixin {
   /// Creates an object that manages the focus tree.
   ///
   /// This constructor is rarely called directly. To access the [FocusManager],
   /// consider using [WidgetsBinding.focusManager] instead.
   FocusManager() {
     rootScope._manager = this;
-    assert(rootScope._firstChild == null);
-    assert(rootScope._lastChild == null);
+    RawKeyboard.instance.addListener(_handleRawKeyEvent);
   }
 
-  /// The root [FocusScopeNode] in the focus tree.
+  /// The root [FocusNode] in the focus tree.
   ///
-  /// This field is rarely used directly. Instead, to find the
-  /// [FocusScopeNode] for a given [BuildContext], use [FocusScope.of].
-  final FocusScopeNode rootScope = FocusScopeNode();
+  /// This field is rarely used directly. Instead, to find the nearest
+  /// [FocusNode] for a given [BuildContext], use [Focus.of].
+  ///
+  /// To find the nearest [FocusNode] that corresponds to a
```

Comment:
This doesn't seem to be saying anything different than the previous sentence?

Reply:
Fixed.